pkgs/extensions/blackfire/1.50.0.nix

```diff
@@ -0,0 +1,57 @@
+{
+  stdenv,
+  pkgs,
+  prev,
+  ...
```

Comment:
This is not needed then.

Reply:
That will be for tomorrow or if you feel like doing it, go for it, feel free.

Reply:
Fixed.